docs/style.css

```diff
@@ -36,12 +50,33 @@
   color: var(--theme-foreground-faint);
 }
 
-.observablehq-link-active a[target=_blank]:not(:hover, :focus)::after {
+.observablehq-link-active a[target="_blank"]:not(:hover, :focus)::after {
   color: var(--theme-foreground-focus);
 }
 
 #observablehq-main {
   min-height: calc(100vh - 23.8rem);
+  font-size: 16px;
+  font-weight: 500;
+}
+
+p a[href] {
+  color: inherit;
+  text-decoration: underline;
+}
+
+h1 {
+  font-family: "Spline Sans Mono", var(--monospace);
```

Comment:
I’d prefer a variable (`--sans-mono`) so we can keep the font definitions together up in `:root`.